third_party/libtock-drivers/src/usb_ctap_hid.rs

```diff
@@ -71,6 +71,24 @@ pub enum UsbInterface {
     VendorHid = 1,
 }
 
+impl UsbInterface {
+  fn from_endpoint(endpoint: usize) -> Option<Self> {
+    match endpoint {
+      1 => Some(UsbInterface::MainHid),
+      #[cfg(feature = "vendor_hid")]
+      2 => Some(UsbInterface::VendorHid),
+      _ => None,
+    }
+  }
+  fn to_endpoint(self) -> usize {
```

Comment:
With the above numbers correctly set, you can remove this function and instead just call:
`as usize`

Reply:
I would not be against keeping the function and just using `as usize` in its implementation. I think the name of the function is good for documentation (unless `UsbInterface` is really just `UsbEndpoint`).